### PR TITLE
Upadte release notes

### PR DESCRIPTION
**Pull request by rtsisyk**
The text was sourced from Etherpad, where it was edited and reviewed by multiple people. This pull request is required to initiate May's release. Please submit your suggestions by the end of the day. I will merge this PR at ~19:00 GMT today.

### CODE REVIEW

**Review by pastk — APPROVED** [review]

File: android/app/src/fdroid/play/listings/ar/release-notes.txt [hunk 1]
```diff
@@ -1,10 +1,8 @@
-• OpenStreetMap login works again!
-• Better isolines elevation data for European countries
-• OpenStreetMap data as of March 26
-• Bookmark lists are sorted by modification time (recent at the top)
-• Eye icons for visible/hidden bookmark lists
-• Fixed crash when importing GPX, KML, KMZ from the Files app
-• Bigger "View search results on the map" button
-• FreeDrive mode and faster bookmarks in Android Auto
+• Upgraded to new OpenStreetMap authentication method - all previously authenticated users must re-login!
```
Comment (biodranik) [inline android/app/src/fdroid/play/listings/ar/release-notes.txt:1]: All logged in OSM users must relogin

File: android/app/src/fdroid/play/listings/ar/release-notes.txt [hunk 2]
```diff
@@ -1,10 +1,8 @@
...
+• OpenStreetMap data as of April 29
+• Display difficult hiking trails as brown dotted, expert ones as black dotted
+• Dark blue lines for dedicated cycleways
+• Blue lines under white/brown dashes for shared foot/cycle paths
```
Comment (biodranik) [inline android/app/src/fdroid/play/listings/ar/release-notes.txt:5]: - sort bookmarks and tracks by name

This is a very important and long-awaited feature.

File: android/app/src/fdroid/play/listings/ar/release-notes.txt [hunk 3]
```diff
@@ -1,10 +1,8 @@
...
+• Roads visible under semi-transparent bridge outlines
```
Comment (biodranik) [inline android/app/src/fdroid/play/listings/ar/release-notes.txt:7]: In Korean description add - Wikipedia articles in the Korean language (can be google translated)

File: android/app/src/fdroid/play/listings/ar/release-notes.txt [hunk 4]
```diff
@@ -1,10 +1,8 @@
...
```
Comment (biodranik) [inline android/app/src/fdroid/play/listings/ar/release-notes.txt:6]: This is a minor change, better to add keywords-friendly and more important one:

- Fixed GPX import from Google Files app

File: android/app/src/fdroid/play/listings/ar/release-notes.txt (hunk 2 above)
Comment (biodranik) [inline android/app/src/fdroid/play/listings/ar/release-notes.txt:5]: All style changes can be shortened:
- Different colors for cycle and hiking paths (long-tap to see more info)

File: iphone/metadata/ar-SA/release_notes.txt [hunk 5]
```diff
@@ -1,22 +1,48 @@
...
+•  Fix circular downloading progress bar layout in selected POIs
+•  Fix radiOn/Off buttons icons
+•  Fix alignment of LTR languages in bottom menu
+•  Correct location of "more" menu in bookmark lists
+•  Add default email client support to about menu
+•  More feature types could be added to OSM now, e.g. marketplaces, taxi stands, kiosks, wilderness huts, garden centers, historic aircraft, cannons, boundary stones, tombs etc.
+•  Show availability of Drivethrough service for selected POIs
+•  Ability to toggle Drivethrough in the OSM editor
+•  Simplify handling of multiple languages feature names in the OSM editor and fix related bugs
+•  "Can't find a suitable category?" in the OSM editor
```
Comment (biodranik) [inline iphone/metadata/ar-SA/release_notes.txt:15]: Help message in the OSM editor when adding a new object

File: iphone/metadata/ar-SA/release_notes.txt [hunk 6]
```diff
@@ -1,22 +1,48 @@
...
-• Many new objects can be added to OpenStreetMap: various shops and crafts, BBQs, food courts, lang schools, cairns, money transfer points, etc.
-• Add "wheelchair" to the search input to limit results to wheelchair-accessible
-• Searchable orders pickup points (shop=outpost)
-• Added a long tap back button menu
 
-Map styling:
-• Added wind turbines and power plant icons
-• Reduced visibilities of many outdoors-specific POIs when in the main map style, e.g., waterfalls, mountain peaks, shelters, power towers, etc. (use the Outdoors style for those!)
-• More consistent font sizes and caption appearances
+•  Upgraded to new OpenStreetMap authentication method (OAuth 2). All previously authenticated users must re• login!
```
Comment (biodranik) [inline iphone/metadata/ar-SA/release_notes.txt:2]: - Remove extra space before the text
- Fix re• login

**Review by biodranik — CHANGES_REQUESTED** [review]

**Comment by pastk** [comment]
> I ran it through ChatGPT...

What for?
I'm concerned it could change the meaning of some points...
At least it should be applied beforehand so that we can proof-read the result.

**Pull request merged**